tests/examples/common.py

```diff
@@ -0,0 +1,63 @@
+import os
+import tempfile
+import subprocess
+
+MOCOCRW_EXAMPLE_BINARY_PATH = "."
+try:
+    MOCOCRW_EXAMPLE_BINARY_PATH = os.environ["MOCOCRW_EXAMPLE_BINARY_PATH"]
+except KeyError:
+    print("Please set the MOCOCRW_EXAMPLE_BINARY_PATH environment variable.")
+    exit(1)
```

Comment:
Why have a default value if you're going to `exit(1)` anyway on error? Also, I think this should be

```python
print(..., file=sys.stderr)
```

and

```python
sys.exit(1)
```